plugin-server/src/main/ingestion-queues/batch-processing/each-batch-async-handlers.ts

```diff
@@ -0,0 +1,21 @@
+import { EachBatchPayload, KafkaMessage } from 'kafkajs'
+
+import { runInstrumentedFunction } from '../../utils'
+import { KafkaQueue } from '../kafka-queue'
+import { eachBatch } from './each-batch'
+
+export async function eachMessageAsyncHandlers(message: KafkaMessage, queue: KafkaQueue): Promise<void> {
+    const event = JSON.parse(message.value!.toString())
```

Comment:
Also this needs significant rework/testing to make sure it works properly off of clickhouse queue. Plus tests, ideally :)